index.html

```diff
@@ -42,46 +34,184 @@ <h2 class="title-about">I'm a Software Developer</h2>
             don’t hesitate to contact me.
         
         </p>
-
-        <div class="deco1"></div>
         
 
         <ul class="social-icon">
             <li>
                 <a href="https://github.com/ulises2607" class="soc-icon">
-                    <img src="Images/Social/GitHub.svg" alt="github-profile">
+                    <img src="Images/Main/Social/GitHub.svg" alt="github-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/" class="soc-icon">
-                    <img src="Images/Social/Linkedin.svg" alt="linkedin-profile">
+                    <img src="Images/Main/Social/Linkedin.svg" alt="linkedin-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.angellist.com/" class="soc-icon">
-                    <img src="Images/Social/Angelist.svg" alt="Angelist-mainpage">
+                    <img src="Images/Main/Social/Angelist.svg" alt="Angelist-mainpage">
                 </a>
             </li>
             <li>
                 <a href="www.twitter.com" class="soc-icon">
-                    <img src="Images/Social/Twitter.svg" alt="twitter-mainpage">
+                    <img src="Images/Main/Social/Twitter.svg" alt="twitter-mainpage">
                 </a>
             </li>
             <li>
                 <a href="https://medium.com/" class="soc-icon">
-                    <img src="Images/Social/Medium.svg" alt="medium-mainpage">
+                    <img src="Images/Main/Social/Medium.svg" alt="medium-mainpage">
                 </a>
             </li>
 
         </ul>
-            
 
-        
+    </section>
+
+    <section class="recent-works">
+
+        <h1>My Recent Works</h1>
+
+        <hr>
```

Comment:
👏👍 Great job on the title! However, it seems that the horizontal line is not displaying in the browser as per the design. Please make sure to update the styling of the horizontal line to ensure it is visible on the webpage. This will help maintain consistency with the design and enhance the overall visual appeal of the title section. Keep up the good work! 😊🎉

![image](https://github.com/ulises2607/Portfolio/assets/55840999/a9ab3537-aa0f-48aa-91d8-3c48fe475adc)